packages/teams-js/test/internal/utils.spec.ts

```diff
@@ -164,4 +171,98 @@ describe('utils', () => {
       expect(createTeamsAppLink(params)).toBe(expected);
     });
   });
+
+  describe('base64ToBlob', () => {
+    it('should convert base64 string to Blob for image/png MIME type', async () => {
```

Comment:
Would be good to have unit tests in here for things like "what if there's no mimetype specified on the data", "What if the string passed in is empty or null" etc.

Reply:
Updated in the latest commit